src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/models/CCJContent.java

```diff
@@ -2,47 +2,57 @@
 
 import uk.gov.hmcts.cmc.claimstore.models.Address;
 import uk.gov.hmcts.cmc.claimstore.models.Claim;
-import uk.gov.hmcts.cmc.claimstore.models.amount.AmountBreakDown;
-import uk.gov.hmcts.cmc.claimstore.models.otherparty.TheirDetails;
+import uk.gov.hmcts.cmc.claimstore.models.CountyCourtJudgment;
+import uk.gov.hmcts.cmc.claimstore.models.otherparty.IndividualDetails;
+import uk.gov.hmcts.cmc.claimstore.services.interest.InterestCalculationService;
+import uk.gov.hmcts.cmc.claimstore.services.staff.content.countycourtjudgment.AmountRemainingContentProvider;
+import uk.gov.hmcts.cmc.claimstore.services.staff.content.countycourtjudgment.RepaymentPlanContentProvider;
 import uk.gov.hmcts.cmc.claimstore.utils.Formatting;
 
 import static java.util.Objects.requireNonNull;
-import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatMoney;
+import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatDate;
 
 public class CCJContent {
 
+    private String requestedAt;
     private String claimReferenceNumber;
     private String claimantName;
     private String requestedDate;
     private String defendantName;
     private Address defendantAddress;
     private String defendantEmail;
     private String amountToPayByDefendant;
-    private String paymentRepaymentOption;
+    private String repaymentOption;
     private String signerName;
     private String signerRole;
+    private String defendantDateOfBirth;
 
-    public CCJContent(Claim claim) {
+    public CCJContent(Claim claim, InterestCalculationService interestCalculationService) {
```

Comment:
When previous comment is addressed then we can just pass `RemainingAmountCalculator ` here, calculate value and format using money formatter.